package.json

```diff
@@ -21,9 +21,13 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:dependencies",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
     "prepack": "./scripts/prepack.sh",
+    "sync-git-hooks": "simple-git-hooks",
     "test": "jest && jest-it-up",
     "test:watch": "jest --watch"
   },
+  "simple-git-hooks": {
+    "pre-push": "yarn lint"
```

Comment:
I would prefer this to be `pre-commit`. If you separate the commands a little bit, it could run `eslint --fix` and `prettier --write` as well, using `lint-staged`. See [snaps-monorepo](https://github.com/MetaMask/snaps-monorepo) for example.

Reply:
If changing to `pre-commit`, maybe we can add a note in `README.md` on the existence of `git commit -n` to make onboarding smoother?

Reply:
I definitely agree with using some kind of mechanism to lint fewer files. That would go a long way in improving the developer experience, I believe, and would possibly remove the concerns I have about using a pre-commit hook. `lint-staged` requires that you use a `.prettierignore` file which I'd like to avoid. I figured out some other way of doing this for another project, however, so I'll see if I can find that.

Reply:
H